src/main/java/io/github/phantamanta44/libnine/client/gui/component/GuiComponentManager.java

```diff
@@ -49,11 +54,26 @@ public boolean handleKeyTyped(char typed, int keyCode) {
         for (ComponentState comp : comps) {
             if (comp.handleKeyTyped(typed, keyCode)) {
                 passThrough = false;
+                break;
             }
         }
+        if (passThrough) {
+            passThrough = focusFirstTextFieldOnKeys(Arrays.asList(Keyboard.KEY_TAB, Keyboard.KEY_NUMPADENTER, Keyboard.KEY_RETURN), keyCode);
+        }
         return passThrough;
     }
 
+    private boolean focusFirstTextFieldOnKeys(List<Integer> keys, int keyCode) {
```

Comment:
what is the `keys` parameter for? it can be factored out
(in fact, this whole method can be inlined, since it only has one call site)

Reply:
For now it's useless. On extending functionality may come in handy